### ISSUE TITLE
Do not calculate scores when calculating a semantic facet

### ISSUE THREAD

**Issue by brent-hartwig** [project-lux/lux-marklogic#341]
**Problem Description:**
To calculate a semantic facet, the facets library uses the search library to search for the facet values.  The search presently defaults to calculating relevancy scores.  There makes the operator slower and scores are not used (facets are sorted by number of times the value is in the search results).  The only way to instruct the search library not to calculate scores is to sort the results by a range index.  We don't want that either.

**Expected Behavior/Solution:**
Two parter:

1. Extend search's support for sort (starting with the endpoint) to neither calculate scores nor sort by anything.  This may only require changes within SortCriteria.mjs.
2. Modify `_getViaSearchFacetValues` within facetsLib.mjs to instruct LUX's search not to calculate scores or sort.

**Requirements:**
See above.

**Needed for promotion:**
If an item on the list is not needed, it should be crossed off but not removed.

~- [ ] Wireframe/Mockup - Mike~
- [ ] Committee discussions - Sarah
- [ ] Feasibility/Team discussion - Sarah
- [x] Backend requirements - See above.
- [x] Middle tier requirements - None.
- [x] Frontend requirements - None.
- [ ] Compels a performance test?  Only if you want but manual testing should be able to prove/quantify individual request improvements (and that we didn't find a way to make the request take longer!).
- [ ] Are new regression tests required for QA - Amy
- [ ] Questions
- List of questions for discussions. Answers should be documented within the issue.

**UAT/LUX Examples:**
The [items curated by YUL](https://lux-front-tst.collections.yale.edu/view/results/objects?q=%7B%22OR%22%3A%5B%7B%22memberOf%22%3A%7B%22curatedBy%22%3A%7B%22id%22%3A%22https%3A%2F%2Flux.collections.yale.edu%2Fdata%2Fgroup%2Fd07b9b86-0a1e-4026-aa4c-8ecba8bbd9c9%22%7D%7D%7D%2C%7B%22memberOf%22%3A%7B%22curatedBy%22%3A%7B%22memberOf%22%3A%7B%22id%22%3A%22https%3A%2F%2Flux.collections.yale.edu%2Fdata%2Fgroup%2Fd07b9b86-0a1e-4026-aa4c-8ecba8bbd9c9%22%7D%7D%7D%7D%5D%7D&qt=objects) search has an estimate of 14.8 million results.  Both semantic facet requests timeout.  Perhaps they won't after this change?  TBD just how much of an impact this change will have.

The above search's criteria:

```json
{
  "OR":[
    {
      "memberOf":{
        "curatedBy":{
          "id":"https://lux.collections.yale.edu/data/group/d07b9b86-0a1e-4026-aa4c-8ecba8bbd9c9"
        }
      }
    },
    {
      "memberOf":{
        "curatedBy":{
          "memberOf":{
            "id":"https://lux.collections.yale.edu/data/group/d07b9b86-0a1e-4026-aa4c-8ecba8bbd9c9"
          }
        }
      }
    }
  ]
}
```

The above criteria is included in the semantic facets' search criteria.

**Dependencies/Blocks:**
This issue neither blocks or is blocked by another ticket.

**Related Github Issues:**
If you want to further improve the performance of this operation, make the search unfiltered (#223).

**Related links:**
None

**Wireframe/Mockup:**
N/A
